datahub-web-react/src/app/ingest/source/builder/RecipeBuilder.tsx

```diff
@@ -65,16 +73,57 @@ function RecipeBuilder(props: Props) {
         }
     }
 
+    let link: React.ReactNode;
+    if (type === 'looker') {
+        link = (
+            <a href={LOOKER_DOC_LINK} target="_blank" rel="noopener noreferrer">
+                DataHub looker module
+            </a>
+        );
+    } else if (type === 'lookml') {
+        link = (
+            <a href={LOOKML_DOC_LINK} target="_blank" rel="noopener noreferrer">
+                DataHub lookml module
+            </a>
+        );
+    }
+
     return (
         <div>
-            <ButtonsWrapper>
-                <StyledButton type="text" isSelected={isViewingForm} onClick={() => switchViews(true)}>
-                    <FormOutlined /> Form
-                </StyledButton>
-                <StyledButton type="text" isSelected={!isViewingForm} onClick={() => switchViews(false)}>
-                    <CodeOutlined /> YAML
-                </StyledButton>
-            </ButtonsWrapper>
+            {(type === 'looker' || type === 'lookml') && (
```

Comment:
I would use constants defined in `src/app/ingest/source/builder/constants.ts` here

Reply:
Checked and fixed!!